django/apps/registry.py

```diff
@@ -5,8 +5,9 @@
 import warnings
 
 from django.core.exceptions import AppRegistryNotReady, ImproperlyConfigured
-from django.utils import lru_cache
+from django.utils import lru_cache, six
 from django.utils.deprecation import RemovedInDjango19Warning
+from django.utils.functional import curry
```

Comment:
I think `django.utils.functional.curry` only still exists for backwards-compatibility (because of the method-binding difference from `functools.partial`) -- in new uses, I think `functools.partial` should be preferred.